html/modules/custom/ocha_docstore_files/ocha_docstore_files.info.yml

```diff
@@ -1,8 +1,5 @@
-name: Flysystem for OCHA document store
-description: 'Provides a plugin for the OCHA documnent store for Flysystem.'
+name: OCHA document store integration
+description: OCHA document store integration
 type: module
 core: 8.x
 core_version_requirement: ^8 || ^9
-package: Flysystem
-dependencies:
- - flysystem:flysystem
```

Comment:
This might need to be a separate PR - I updated everything but couldn't import config as flysystem, upgrade_rector and upgrade_status were missing. I then went back to `develop`, and php7.4, but need to uninstall ocha_docstore_files to be able to uninstall flysystem. (Which is okay on local, but could get sticky on staging and production.)

![image](https://user-images.githubusercontent.com/67453/131830725-473d04ec-ab00-4d48-8f20-a02e27f9c0ff.png)